### ISSUE TITLE
About Loss

### ISSUE THREAD

**Issue by Fleurrr** [marvin-eisenberger/implicit-sinkhorn#1]
Why is loss a negative number, is this normal?
![image](https://user-images.githubusercontent.com/25102449/180965733-e1d3bff9-181b-45d9-93c6-f20675133fac.png)

**Comment by marvin-eisenberger** [comment]
Hi, this is indeed normal. The second term of the loss (the entropy regularizer) commonly has values <0. 

The exact term is specified here:
https://github.com/marvin-eisenberger/implicit-sinkhorn/blob/b6e5499d51864bd41e36aeb07b7d889c77d16a8c/image_barycenter/image_wasserstein.py#L58

**Comment by Fleurrr** [comment]
Thanks for your answer!
For example, I want to optimize this loss with other positive losses. Obviously, it is unreasonable to accumulate them as Total_loss = loss_sinkhorn(-) + loss_other(+). 
How should I modify your loss function?

**Comment by marvin-eisenberger** [comment]
I do not see an issue there, simply adding the two loss terms should work, regardless of whether loss_sinkhorn(-) or loss_sinkhorn(+).
